script.js

```diff
@@ -34,10 +37,9 @@ function displayBooks(books) {
     let publisher = volumeInfo.publisher || "N/A";
     let smallThumbnail = getThumbnail(volumeInfo);
 
-    let bookDiv = document.createElement("div");
+    let $bookDiv = $("<div>").addClass("book-styling");
```

Comment:
The jQuery syntax for creating new DOM nodes is ... horrible. It works well enough for a quick addition, like this `<div>`, but for more than that, it gets unwieldly.